src/modules/account/components/RemoveConfirmation/RemoveConfirmation.js

```diff
@@ -22,29 +25,22 @@ const RemoveConfirmation = ({
   return (
     <>
       <h1>{t('Remove Account?')}</h1>
-      <WalletVisual
-        className={styles.avatar}
-        address={account?.metadata?.address}
-      />
-      {
-        account?.metadata?.name
-        && (<p className={styles.accountName}>{account?.metadata?.name}</p>)
-      }
+      <WalletVisual className={styles.avatar} address={account?.metadata?.address} />
+      {account?.metadata?.name && <p className={styles.accountName}>{account?.metadata?.name}</p>}
```

Comment:
We can use the local variable declared at the top, `accountName`, to prevent us from using two different ones for the same data.

Reply:
@oskarleonard Resolved